models/articles-models.js

```diff
@@ -21,4 +21,15 @@ exports.selectArticleById = (param_id) => {
         if (rows.length === 0) return Promise.reject({status: 404, msg: "article does not exist"})
         return rows[0];
     })
+}
+
+exports.selectArticleComments = (param_id) => {
+    return db.query(`
+    SELECT * FROM comments
+    WHERE article_id = $1
+    ORDER BY created_at DESC
+    `, [param_id]).then(({rows}) => {
```

Comment:
See comment re: testing - if the ID is valid and the article exists but there are no comments for the article, we just want to return an empty array